numpy/_core/src/umath/string_fastsearch.h

```diff
@@ -29,13 +29,36 @@
    algorithm, which has worst-case O(n) runtime and best-case O(n/k).
    Also compute a table of shifts to achieve O(n/k) in more cases,
    and often (data dependent) deduce larger shifts than pure C&P can
-   deduce. See stringlib_find_two_way_notes.txt in this folder for a
-   detailed explanation. */
+   deduce.*/
```

Comment:
Maybe add a note to see the file in CPython instead. Something like `See Objects/stringlib/stringlib_find_two_way_notes.txt` in the CPython repository for a detailed explanation.`

Reply:
I agree, adding a reference would be helpful. I'll include a hyperlink to the CPython repository.